packages/core/lib/services/credential/service.js

```diff
@@ -44,6 +46,19 @@ class CredentialService {
 
     return verifiableCredential;
   }
+  async createPresentation(params) {
+    validation.createPresentation(params);
+    const {credentials, keyDoc, challenge, id} = params;
+    const vp = new VerifiablePresentation(id);
+    for (const signedVC of credentials) {
+      vp.addCredential(signedVC);
+    }
+    const kp = getKeypairFromDoc(keyDoc);
+    kp.signer = kp.signer();
+    const suite = getSuiteFromKeyDoc(kp);
+    vp.setHolder(keyDoc.controller);
+    return vp.sign(suite, challenge);
```

Comment:
`params` should have `domain` and `resolver` args as well, https://github.com/docknetwork/sdk/blob/master/src/verifiable-presentation.js#L173